README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse-project
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Hello Microverse project]
+
+**[Hello Microverse project]** is a simple project that display a hello microverse message.
+
+## 🛠 Built With
+
+### Tech Stack
+
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
+
+### Key Features
+
+- **[When clicked it prints "Hello Microverse" in green color]**
+
+## 🚀 Live Demo
+
+- [No Live Demo Link]
+
+## 💻 Getting Started
+
+prints Hello Microverse in default browser.
+
+### Prerequisites
+
+Clone this repository to your desired folder and open html file.
+
```

Comment:
[OPTIONAL] Well done on the `README` 👏🏽 you used the professional one as requested by Microverse but for these two sections, it would be better to add instructions on how to clone the project. You could add for example a line like this. 👇🏽 
```
git clone https://github.com/ntimanu/hello-microverse-project.git
```